core/deployment/src/main/java/io/quarkus/deployment/builditem/DevServicesSharedNetworkBuildItem.java

```diff
@@ -81,4 +82,17 @@ public static boolean isSharedNetworkRequired(
                 (!devServicesSharedNetworkBuildItem.isEmpty()
                         && devServicesSharedNetworkBuildItem.get(0).getSource().equals("io.quarkus.test.junit"));
     }
+
+    /**
+     * @deprecated Please, use {@link DevServicesSharedNetworkBuildItem#isSharedNetworkRequired(DevServicesConfig, List)}
+     *             instead.
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
Please add a `since = "3.18"` so that we have an indication of when it's safe to remove it.

Thanks!

Reply:
Done.